src/_data/how_to/get_connected.yml

```diff
@@ -0,0 +1,46 @@
+toc:
+  - title: Getting started
+    subsection_items:
+      - title: What is connectivity?
+        url: "what-is-connectivity"
+        paragraphs:
+          - text: From accepting mobile wallets for fare payments to giving the live location of a bus, transit is using more technology to improve the rider experience. Cellular data is the power behind this new technology. Types of devices may include&#58;
+          - text: <ul><li>Router</li><li>CAD/AVL</li><li>Camera feeds</li><li>Passenger WiFi</li></ul>
+          - text: As transit continues to modernize, more devices can be expected to leverage cellular data and connectivity.
+      - title: How do data plans work?
+        url: "how-do-data-plans-work"
+        paragraphs:
+          - text: Your technology can be connected through multiple SIMs (one per device) or a single SIM (one in a router).
+          - text: <u>Multiple SIMs</u>
```

Comment:
I feel like these would be more appropriate as `<h4>` styled as the underlined text.

See the [Usage Notes](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/u#usage_notes) on the `<u>` element, it doesn't seem right here:

> You should _not_ use `<u>` to underline text for presentation purposes, or to denote titles of books.

Reply:
Resolved: https://github.com/cal-itp/mobility-marketplace/pull/406/commits/4a3dbcae53d9387eb8340d3f3bdb80a36555f1ba